blocks/subscriptions-block/components/ContactInfo/index.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useRef } from "react";
+
+import { usePhrases, Heading, Input, Button, Stack, Image, Icon } from "@wpmedia/arc-themes-components";
+
+import GoogleIcon from "./google.svg";
+
+const ContactInfo = ({ callback, user, signedInIdentity, logoutCallback, className }) => {
+	const formRef = useRef();
+	const entriesRef = useRef({});
+
+	if (user) {
+		entriesRef.current.email = user.email ? user.email : "";
+		entriesRef.current.firstName = user.firstName ? user.firstName : "";
+		entriesRef.current.lastName = user.lastName ? user.lastName : "";
+	}
+
+	const handleSubmit = (event) => {
+		event.preventDefault();
+		const valid = formRef.current.checkValidity();
+		if (valid) {
+			callback(entriesRef.current);
+		}
+	};
+
+	const handleInputChange = (name, entry) => {
+		entriesRef.current[name] = entry.value;
+	};
+
+	const handleLogout = () => {
+		logoutCallback();
+	};
+
+	const phrases = usePhrases();
+
+	// TO-DO: Uncomment once Select component is done
+	// const getTranslatedCountries = countryCodes.map((entry) => ({
+	// 	code: entry.code,
+	// 	name: phrases.t(entry.key),
+	// }));
+
+	const signOutBtn = (
+		<Button variant="secondary" size="medium" onClick={handleLogout}>
+			{phrases.t("checkout-block.identity-sign-out")}
+		</Button>
+	);
+
+	const identityInfo = () => {
+		if (user && signedInIdentity) {
+			const type = signedInIdentity.type.toLowerCase();
+			switch (type) {
+				case "google":
+					return (
+						<div className={`${className}__identity-row`}>
+							{/* {TO-DO: Change the Icon to Google <Icon name="Google"/> } */}
+							<Image alt="Google" title="Google" src={GoogleIcon} />
+							<span>
+								{phrases.t("checkout-block.identity-social", {
+									email: user.email,
+								})}
+							</span>
+							{signOutBtn}
+						</div>
+					);
+				case "facebook":
+					return (
+						<div className={`${className}__identity-row`}>
+							<Icon name="Facebook"/>
```

Comment:
@LauraPinilla , should be a quick update, but you will want to set the `fill` prop here, with the Facebook blue (I think it's `#415993`).

Reply:
As I'm working on the icons ticket, I'm noticing this prop is not functioning. You can set the fill via CSS (with a token specific to this Facebook icon) as a workaround for now.